sdks/python/apache_beam/runners/worker/statesampler_fast.pyx

```diff
@@ -109,25 +80,18 @@ cdef class StateSampler(object):
   cdef pythread.PyThread_type_lock lock
 
   cdef public int64_t state_transition_count
-  cdef int64_t time_since_transition
+  cdef int64_t _time_since_transition
```

Comment:
Why this change? Or should we be changing all the others as well for consistency?